packages/now-build-utils/test/unit.builds-and-routes-detector.test.ts

```diff
@@ -1105,7 +1145,19 @@ it('Test `detectRoutes` with `featHandleMiss=true`', async () => {
       builders!,
       featHandleMiss
     );
-    expect(defaultRoutes).toStrictEqual(expectedRoutes);
+    expect(defaultRoutes).toStrictEqual([
+      { handle: 'miss' },
+      {
+        src: '^/api/([^\\/]+)$',
+        dest: '/api/[endpoint].js?endpoint=$1',
+        check: true,
+      },
+      {
+        status: 404,
+        src: '/api(/.*)?$',
```

Comment:
Can we add a `^` to the begining of this so it doesn't need to be normalized

Reply:
Thanks, I fixed it 👍